src/libcore/any.rs

```diff
@@ -446,14 +446,16 @@ impl TypeId {
 /// # Note
 ///
 /// This is intended for diagnostic use. The exact contents and format of the
-/// string are not specified, other than being a best-effort description of the
-/// type. For example, `type_name::<Option<String>>()` could return the
-/// `"Option<String>"` or `"std::option::Option<std::string::String>"`, but not
-/// `"foobar"`. In addition, the output may change between versions of the
-/// compiler.
+/// string retrned are not specified, other than being a best-effort description
```

Comment:
`returned`?

Reply:
Mea culpa! Fixed in https://github.com/rust-lang/rust/pull/71870/commits/876001c8ac331a15661638fbb18b8b64facf2c96.